en-ar/jw300-tedtalk-baseline/notebook.ipynb

```diff
@@ -0,0 +1,1370 @@
+{
```

Comment:
<p>The one thing we need to ensure is that none of the sentences in the "tedtalk" dataset occur in the JW300 test set. </p><p>I doubt they have any overlap, but would you mind running this check?  </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/masakhane-io/masakhane/pull/59/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='27'/>

Reply:
<p>i will run a test in  the notebook for this </p>

Reply:
resolved